<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Remote start</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of tickets for every ticket type:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Prc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Params">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LotteryTicketsNumber 1 50 LotteryTicketsNumber 2 40 LotteryTicketsNumber 3 30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Percentage of winning tickets:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Send">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send start command</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Factory full work time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_WorkTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0 seconds</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Network params</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Port">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server IP address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_IP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>281</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Winning tickets</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>261</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GetWinTick">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Get winning tickets</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TicketID - TicketType</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
